gradle/libs.versions.toml

```diff
@@ -31,25 +30,13 @@ slf4j-mock = "org.simplify4u:slf4j-mock:2.3.0"
 #      (1.*.*, rather than a specific version like 1.7.36).
 slf4j-simple = "org.slf4j:slf4j-simple:1.7.36"
 vavr = "io.vavr:vavr:0.10.4"
-# Plugin packages
```

Comment:
hmmm shouldn't this comment be at the top? 🤔 anyway pls provide comments here and in the other `libs.versions.toml` that describe what should be defined and where - now it's a little confusing to me 😅

Reply:
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/19799111/218026268-a0e2dc4e-0ac3-47b4-9682-03d2ee90e5eb.png">
 
unrel have you ever seen that? XD

Reply:
> unrel have you ever seen that? XD

Yeah, a bug in github... it's probably coz this file used to be a symlink but has been replaced with a regular file. This is always a nasty corner case :/

Reply:
> hmmm shouldn't this comment be at the top? 

It's right before `pluginPackages-...` dependencies... a proper place!

Reply:
> anyway pls provide comments here and in the other libs.versions.toml that describe what should be defined and where

Lemme explain a few non-trivial points... which should be enough for anybody interested to decipher the whole picture! (so as to avoid explaining each and every dependency, as TBH almost each has a slightly different use case, esp. in buildSrc/ 😅)